src/app/callback/callback.component.html

```diff
@@ -0,0 +1,2 @@
+<script>
```

Comment:
Do we need the empty script tags?

Reply:
No, those are remnants of the callback that have been moved into the `NgOnInit()` method in callback.component.ts. They wont hurt anything but I'll remove them.